code/API_definitions/geofencing.yaml

```
@@ -43,7 +43,11 @@ info:
     
     Note: Additionally to these list, ``org.camaraproject.geofencing.v0.subscription-ends`` notification `type` is sent when the subscription ends. 
     This notification did not requires dedicated subscription. 
-    It is used when the subscription expire time (required by the requester) has been reached or if the API server has to stop sending notification prematurely. 
+
+    It is used when:
+      - the subscription expire time (required by the requester) has been reached 
```

This change will be in conflict with #162. I suggest to consolidate both changes here and close the other one @maxl2287 

```suggestion
      - the subscription expire time (optionally set by the requester) has been reached 
```